docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst

```diff
@@ -145,13 +162,59 @@ The sub-aim for this project is to create a 'power calculation' of how many pati
 4.0 Intervention
 ++++++++++++++++
 
-
+Among MM and RRMM patients, they expect to have:
+ - Isatuximab treatment with coverage scale-up from 10% to 45% across 5 years 
```

Comment:
Perhaps we should dial this down substantially, to start from 1% instead of 10; let's discuss.